archesproject/arches-references
# Add primevue 4.1 compat

- Remove ugly workaround for lazy filtering on labels in selected language
- Cope with primevue 4.1 class name change

### Testing instructions
- Save labels in multiple languages
- Switch the "show labels in:" language selection inside the tree
- Label filtering should still search on the selected language
- Bolding of search results still works
